client/src/index.scss

```diff
@@ -0,0 +1,100 @@
+$light-grey: rgb(230, 230, 230);
+$medium-grey:rgb(137, 137, 137);
+$dark-grey: rgb(76, 76, 76);
+.reviews-section-container {
+  @extend .bottom-padding;
+  font-family: Arial;
+  font-size: 15px;
+  color: black;
+  .reviews-section-title {
+    @extend .bottom-margin;
+    font-weight: 600;
+    line-height: 2rem;
+    color: black;
+  }
+
+  .reviews-section-stats{
+    @extend .bottom-margin;
+    @extend .light-border-bottom;
+    @extend .bottom-padding;
```

Comment:
I like to use `@extend` for inheritance of multiple CSS properties. Using it for single properties like this makes me wonder if CSS variables would be a better use case. Curious what you think.

Reply:
i like variables for repeated values across properties, but for repeating a set property i make a utitlity class. i see how it might be a overkill for a single value though.